tox.ini

```diff
@@ -44,56 +46,79 @@ setenv =
     # make sure that directory is accessible by docker
     hive: UPLOAD_FOLDER = /tmp/.superset/app/static/uploads/
 usedevelop = true
-whitelist_externals =
+allowlist_externals =
     npm
+    pkill
 
-[testenv:cypress-dashboard]
+[testenv:cypress]
+setenv =
+    PYTHONPATH = {toxinidir}
+    SUPERSET_TESTENV = true
+    SUPERSET_CONFIG = tests.superset_test_config
+    SUPERSET_HOME = {envtmpdir}
+    ENABLE_REACT_CRUD_VIEWS = true
 commands =
     npm install -g npm@'>=6.5.0'
     pip install -e {toxinidir}/
-    {toxinidir}/superset-frontend/cypress_build.sh dashboard
-deps =
-    -rrequirements/testing.txt
+    {toxinidir}/superset-frontend/cypress_build.sh
+commands_post =
+    pkill -if "python {envbindir}/flask"
+
+[testenv:cypress-dashboard]
```

Comment:
Oh interesting, we still have the tox envs split up as 3 different suite (we use to run 3 cells in Travis), but on GitHub actions we're running it inside a single cell inside the build matrix.

NOTE: We discussed in the past whether we want to commit or break away from `tox` as a way to run local tests. But that's outside the scope of this PR!

Reply:
I considered using tox for local parallelization, which I can revisit in a separate PR.